pkg/opensearch/resource_handlers.go

```diff
@@ -0,0 +1,61 @@
+package opensearch
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+// handleRegisterStreamQuery handles the HTTP request to register a query for a streaming session.
```

Comment:
I'm still trying to understand the flow here, can you clarify why this step is necessary? I'm looking at [this guide](https://grafana.com/developers/plugin-tools/tutorials/build-a-streaming-data-source-plugin) to enable streaming in datasource plugins
and I don't see this step to register backend stream, it seems like `getGrafanaLiveSrv().getDataStream({` on the frontend is enough.